priv/repo/migrations/20180904222223_create_scenes_table.exs

```diff
@@ -8,15 +8,15 @@ defmodule Ret.Repo.Migrations.CreateScenesTable do
       add(:slug, :string, null: false)
       add(:name, :string, null: false)
       add(:description, :string)
-      add(:author_account_id, :bigint, null: false)
-      add(:model_upload_id, :bigint, null: false)
-      add(:screenshot_upload_id, :bigint, null: false)
-      add(:attribution_name, :string, null: false)
-      add(:attribution_link, :string)
```

Comment:
Leaving out attribution for now?

Reply:
yeah, I took this out since I wasn't sure where we'll land for this. if spoke is more about mixing existing web content i'm thinking there will be a list of attributions we have to crawl out of the GLTFs involved